src/centralserver/admin-service/core/src/test/java/org/niis/xroad/centralserver/restapi/service/OcspRespondersServiceImplTest.java

```diff
@@ -157,6 +160,16 @@ void updateOnlyUrl() {
         assertAuditMessages(ocspInfo, newUrl);
     }
 
+    @Test
+    void shouldThrowExceptionWhenOcspInfoNotFound() {
+        when(ocspInfoRepository.findById(ID)).thenReturn(Optional.empty());
+
+        Executable testable = () -> service.delete(ID);
+
+        NotFoundException actualThrown = assertThrows(NotFoundException.class, testable);
+        assertEquals(CERTIFICATION_SERVICE_NOT_FOUND.getDescription(), actualThrown.getMessage());
+    }
+
```

Comment:
Maybe successful delete unit test would be useful also? It could also verify if the auditing.